### PR TITLE
[BugFix][seatunnel-api] fix bug-6201 After submitting a task to the dev branch through st web an error is reported with inconsistent serialization IDs

### PR DESCRIPTION
**Pull request by gitfortian**
…, an error is reported with inconsistent serialization IDs

### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Review by liugddx — DISMISSED** [review]
LGTM

**Comment by liugddx** [comment]
@Hisoka-X @ruanwenjun  PTAL

File: seatunnel-api/src/main/java/org/apache/seatunnel/api/table/catalog/MetadataColumn.java [hunk 1]
```diff
@@ -27,6 +27,7 @@
 @EqualsAndHashCode(callSuper = true)
 @ToString(callSuper = true)
 public class MetadataColumn extends Column {
+    private static final long serialVersionUID = -1L;
```
Comment (Hisoka-X) [inline seatunnel-api/src/main/java/org/apache/seatunnel/api/table/catalog/MetadataColumn.java]:
```suggestion
    private static final long serialVersionUID = 1L;
```

File: seatunnel-api/src/main/java/org/apache/seatunnel/api/table/catalog/PhysicalColumn.java [hunk 2]
```diff
@@ -30,6 +30,8 @@
 @ToString(callSuper = true)
 public class PhysicalColumn extends Column {
 
+    private static final long serialVersionUID = -1L;
```
Comment (Hisoka-X) [inline seatunnel-api/src/main/java/org/apache/seatunnel/api/table/catalog/PhysicalColumn.java]:
```suggestion
    private static final long serialVersionUID = 1L;
```

**Review by Hisoka-X — APPROVED** [review]

**Review by liugddx — APPROVED** [review]
LGTM

**Comment by liugddx** [comment]
Wait for the CI

**Pull request merged**